tools/cli.js

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env node
+
+'use strict';
+
+// TODO: support jstp://server and jstps://server
+
+const jstp = require('.');
+const readline = require('readline');
+
+const commandProcessor = {};
+const lineProcessor = {};
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  completer
+});
+
+const log = msg => {
```

Comment:
not a problem, should we add it to eslint ([rule](http://eslint.org/docs/rules/arrow-parens))?